test/pamela/HTN/parallel-constraints.main.tpn.edn

```diff
@@ -1,168 +1,264 @@
 {:act-104
- {:args ["host0301" "host0501" "1"],
-  :argsmap {"client" "host0301", "display" "1", "server" "host0501"},
+ {:args
+  [{:type :method-arg-ref, :names [loc2]}
+   {:type :method-arg-ref, :names [loc3]}
+   "1"],
+  :argsmap
```

Comment:
(I should have added the previous comment **after** the new version of the generated TPN snippet.  This is an empty comment to facilitate readability in the Github Conversation view)